andela/suyabay
# #115673279 Fix broken images

#### What does this PR do?

Get user avatar from  gravatar if the link to the uploaded avatar points to an image that was deleted or removed
#### Description of Task to be completed?
- Check if link to avatar is broken
- Get avatar from gravatar website if link is broken
- Display user avatar if the link to the avatar on Cloudinary points to an existent image
#### How should this be manually tested?
- Change you avatar
- Head over to Cloudinary and delete the image you just uploaded as your avatar
- When you visit the dashboard you should see the default gravatar as your avatar
#### What are the relevant pivotal tracker stories?

 #115673279
#### Screenshots (if appropriate)

Before
<img width="284" alt="screen shot 2016-03-15 at 12 19 01 pm" src="https://cloud.githubusercontent.com/assets/8840187/13772998/282772a0-eaa8-11e5-97d1-1c4877435528.png">

After
<img width="273" alt="screen shot 2016-03-15 at 12 20 50 pm" src="https://cloud.githubusercontent.com/assets/8840187/13773068/7df576c8-eaa8-11e5-9a16-62f4f470016e.png">
